lib/takeaway.rb

```diff
@@ -0,0 +1,38 @@
+require_relative "dish"
+require_relative "order"
+require_relative "menu"
+
+class Takeaway
+
+  attr_reader :order, :total, :menu
+
+  def initialize(order = Order.new, menu = Menu.new )
+    @order = order
+    @menu = menu
+    @total = 0
+  end
+
+  def see_menu
+    menu.menu
```

Comment:
This is one of the problems with naming an attribute the same as its class: what would `menu.menu` mean in plain English?